code/datums/emergency_calls/emergency_call.dm

```diff
@@ -66,10 +66,10 @@
 		to_chat(M, "<span class='attack'>You cannot join if you have Ghosted before this message.</span><br>")
 
 
-/datum/game_mode/proc/activate_distress()
-	picked_call = get_random_call()
+/datum/game_mode/proc/activate_distress(datum/emergency_call/chosen_call = null)
+	picked_call = chosen_call || get_random_call()
 
-	if(!picked_call) //Something went horribly wrong
+	if(!istype(picked_call)) //Something went horribly wrong
```

Comment:
`get_random_call()` should be unable to return a wrong type. But if it does, then something indeed went horribly wrong and this could use a `CRASH()` message instead of a silent error.

Or just remove the check entirely.

Reply:
No CRASHes on my watch :p

Reply:
No Crashes on my watch >:)

Reply:
No silent checks on mine.

Reply:
No check = no silent check